freeipa-client/src/main/java/com/sequenceiq/freeipa/client/FreeIpaClient.java

```diff
@@ -197,6 +197,11 @@ public User userSetPasswordHash(String user, String hashedPassword,
         return userMod(user, params);
     }
 
+    public boolean hasSetPasswordHashSupport() throws FreeIpaClientException {
```

Comment:
doesn't quite feel right in the FreeIpaClient class. We will also likely start checking other object classes as well.

I suggest pulling this out into a separate class. Maybe FreeIpaObjectClasses or FreeIpaCapabilities

Reply:
Moved it into a FreeIpaCapabilities class. I assume you meant we still want it in the freeipa client package.